common/workunit/workunit.hpp

```diff
@@ -1689,7 +1690,11 @@ extern WORKUNIT_API bool isValidMemoryValue(const char * memoryUnit);
 
 #define HourToSeconds(n) ((n)/3600)
 #define NanoSecondsToSeconds(n) ((double)(n)/1000000000)
-inline __int64 calcCost(double ratePerHour, __int64 timeNS) { return HourToSeconds(ratePerHour) * NanoSecondsToSeconds(timeNS) * 1e6; }
+#define MilliSecondsToSeconds(n) ((double)(n)/1000)
+inline unsigned __int64 calcCostSecs(double ratePerHour, double secs) { return HourToSeconds(ratePerHour) * secs * 1e6; }
+inline unsigned __int64 calcCostNs(double ratePerHour, unsigned __int64 timeNS) { return calcCostSecs(ratePerHour, NanoSecondsToSeconds(timeNS)); }
+inline unsigned __int64 calcCostMs(double ratePerHour, unsigned __int64 timeMS) { return calcCostSecs(ratePerHour, MilliSecondsToSeconds(timeMS)); }
```

Comment:
do we ever need to calc. cost at NS precision ? Seems unlikely/excessive.
I would have thought per second would suffice for the foreseeable future.